### PR TITLE
fixup .editorconfig

### PR DESCRIPTION
**Pull request by bartimaeusnek**
Adds missing properties to .editorconfig (automatically detected by Rider, non-standard only)
fixes broken properties

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -45,10 +125,10 @@ dotnet_diagnostic.IDE0055.severity = warning
...
 dotnet_separate_import_directive_groups = false
 # Avoid "this." and "Me." if not necessary
-dotnet_style_qualification_for_field = false:refactoring
-dotnet_style_qualification_for_property = false:refactoring
-dotnet_style_qualification_for_method = false:refactoring
-dotnet_style_qualification_for_event = false:refactoring
+dotnet_style_qualification_for_field = false:suggestion # refactoring is not supported by all IDEs
+dotnet_style_qualification_for_property = false:suggestion
+dotnet_style_qualification_for_method = false:suggestion
+dotnet_style_qualification_for_event = false:suggestion
```
Comment (Cazzar) [inline .editorconfig:131]: Are we able to bump this up from a suggestion to something higher - @da3dsoul thoughts?

Comment (da3dsoul) [reply .editorconfig:131]: I think suggestion is one below warning. It'll be underlined yellow, but it doesn't build as a warning

Comment (da3dsoul) [reply .editorconfig:131]: https://learn.microsoft.com/en-us/visualstudio/code-quality/use-roslyn-analyzers?view=vs-2022
That is correct according to this

Comment (Cazzar) [reply .editorconfig:131]: Yeah, I would possibly be pushing it up to a warning if anything, Error seems _too_ strict right now

Comment (da3dsoul) [reply .editorconfig:131]: I'm not against it

**Pull request merged**